packages/x-license-pro/src/generateLicense/generateLicense.ts

```diff
@@ -1,19 +1,30 @@
 import { md5 } from '../encoding/md5';
 import { base64Encode } from '../encoding/base64';
 
-const licenseVersion = '1';
+const licenseVersion = '2';
```

Comment:
Is there any interest to keep the old algorithm available to be able to generate licenses of version 1 ? 

Something like `generateLicense/generateLicenseV1.ts` with a copy paste of the `generateLicense/generateLicense.ts` as it is before this PR

Reply:
I think the reason was so that we can check the version of the license if we need to do some specific checks but I don't think there is a value in keeping the old algorithm.